bzz/incubator-zeppelin
# highlight syntax and search terms in search results

<img width="720" alt="screen shot 2015-11-26 at 11 38 45 am" src="https://cloud.githubusercontent.com/assets/5688346/11413936/718299ec-9432-11e5-9005-c68c6659d104.png">

Test plan:
1. Search for a term
2. See the search results
3. Navigate to paragraphs from search results
